### DISCUSSION TITLE
Retrieving many to many relationship with additional Fields.

### DISCUSSION THREAD

**Discussion by AleksandrukTad** [prisma/prisma#2246] [Q&A] [answered]
Hey just have you came across situation that you have many to many relationship with additonal fields? This is my schema:

```
model Medicine {
  id                    Int      @default(autoincrement()) @id
  name                  String
}

model ActiveSubstance {
  id   Int    @default(autoincrement()) @id
  name String
}

model MedicineActiveSubstance {
  id                      Int             @default(autoincrement()) @id
  medicine                Medicine        @relation(fields: [medicineId], references: [id])
  medicineId              Int
  activeSubstance         ActiveSubstance @relation(fields: [activeSubstanceId], references: [id])
  activeSubstanceId       Int
  amountOfActiveSubstance Int
}
```
this is my nexus:

```
export const Medicine = objectType({
  name: 'Medicine',
  definition(t) {
    t.model.id()
    t.model.name()
  },
})

export const ActiveSubstance = objectType({
  name: 'ActiveSubstance',
  definition(t) {
    t.model.id()
    t.model.name()
  },
})
```

I can add t.model.medicineActiveSubstances (to activeSubstances) and t.model.medicineActiveSubstances (to medicines). But this causes an error when I try to add type to it:
"ERROR: No subset types are available. Please make sure that one of your GraphQL type is a subset of your t.model('<ModelName>')",

How do I retrieve medicine with all ActiveSubstances on it and amountOfActiveSubstance??

**Comment by AleksandrukTad** [comment, marked as answer]
"ERROR: No subset types are available. Please make sure that one of your GraphQL type is a subset of your t.model('')", - This error appears because you name in Nexus model does not match prisma model. As soon as i matched them it works fine. 
I hope this will help others that will come across this issue!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
